projects/step-frontend/src/lib/modules/timeseries/time-series.service.ts

```diff
@@ -3,14 +3,21 @@ import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { FindBucketsRequest } from './find-buckets-request';
 import { TimeSeriesChartResponse } from './time-series-chart-response';
-import { AsyncTaskStatusResource, Execution } from '@exense/step-core';
+import { AsyncTaskStatusResource, AuthService, Execution } from '@exense/step-core';
+import { TimeSeriesConfig } from './time-series.config';
 
 // TODO provide it it TimeSeriesModule
 @Injectable({
   providedIn: 'root',
 })
 export class TimeSeriesService {
-  constructor(private http: HttpClient) {}
+  private readonly resolutionPeriod: number; // this is received from the backend
```

Comment:
As this field is readonly and assigned in the contstructor, `private` modifier and `getResolution` method looks redundant